joomla/joomla-cms
# [4.0] Expand the allowed list for the HTML sanitiser

Pull Request for Issue # .

### Summary of Changes
By default, the sanitiser will remove all the form elements. This PR changes this behaviour so form elements would not disappear (on* events will still be removed).
The list of attributes comes directly from the MDN: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attributes

### Testing Instructions
Apply the PR
Goto admin dashboard and open the browser console
pass some HTML to the sanitiser and check the result:

```js
Joomla.sanitizeHtml(`<input type="text" onchange="alert('nope')" >`);
```
Check more attributes and elements (button, select, textarea)

### Actual result BEFORE applying this Pull Request
Too strict

### Expected result AFTER applying this Pull Request
Less strict


### Documentation Changes Required
